update pricing doc for crons (#8773)
getsentry/sentry-docs: src/docs/product/accounts/quotas/manage-crons-monitors.mdx

---
title: Manage Your Cron Monitors
keywords: ["best practices"]
sidebar_order: 70
description: “Learn how to use Sentry's tools to manage the amount and types of cron monitors you pay for.”
---

All Sentry plans include 1 cron monitor. To activate additional monitors, you need to add [on-demand](/product/accounts/pricing/#on-demand-capacity) spend.

<Note>

On-demand spend is only available on paid plans.

</Note>

> ## Before you begin, check your available on-demand spend

Reviewer: ```suggestion
## Checking Available On-demand Spend
```

## Activating Additional Monitors

Additional monitors will be activated upon creation if there is available on-demand spend. Alternatively, inactive monitors can be manually activated when on-demand spend is availalbe.

> Active monitors will remain active across billing periods if on-demand spend is available.

Reviewer: ```suggestion
Any additional monitors you create will be activated by default as soon as they're created if there's available on-demand spend. Alternatively, any inactive monitors can be manually activated when on-demand spend becomes available.

Active monitors will remain active across billing periods if on-demand spend is available.
```

## Deactivating Monitors

> Monitors can be manually deactivated at anytime from the **Crons** page.

Reviewer: ```suggestion
Monitors can be manually deactivated from the **Crons** page at any time.
```

> All monitors are automatically deactivated at the start of a new billing period if on-demand spend is reduced or not available to cover all active monitors. Monitors can be manually activated based on on-demand availability.

Reviewer: ```suggestion
All monitors will be automatically deactivated at the start of a new billing period if on-demand spend is reduced or not available to cover all active monitors. Monitors can be manually activated based on on-demand availability.
```
